cmake/Modules/OpmStaticTargets.cmake

```diff
@@ -7,15 +7,23 @@
 
 # Macros
 function(opm_from_git repo name revision)
+  if(ARGN)
+    set(COMMANDS BUILD_COMMAND ${CMAKE_COMMAND} --build <BINARY_DIR> --target ${ARGN}
+                 INSTALL_COMMAND)
```

Comment:
> In principle one alternative would be to just build straight out of the source tree but then cmake bugs bites.

I see.  Well, as long as there is a reason--which I suppose we could document in a comment--I'm happy to leave well enough alone.